rpc/rpc_client.go

```diff
@@ -27,7 +27,7 @@ type ClientOpts struct {
 }
 
 type Inserter interface {
-	Insert(ts time.Time, dims map[string]interface{}, vals func(func(string, interface{}))) error
+	Insert(ts time.Time, dims map[string]interface{}, vals func(func(string, float64))) error
```

Comment:
Drive-by, being a bit more strict in what gets passed to this interface function.